src/runtime/threads/policies/hwloc_topology_info.cpp

```diff
@@ -92,6 +93,15 @@ namespace hpx { namespace threads
     }
 
     ///////////////////////////////////////////////////////////////////////////
+    std::ostream& operator<<(std::ostream& os, hpx_hwloc_bitmap_wrapper const* bmp)
+    {
+        char buffer[256];
```

Comment:
Is this buffer large enough for a large number of PUs? Is the generated string hex formatted or binary?

Reply:
yes, it's a hex output. 256 chars should be more than enough for any multi-core chips we see for the foreseeable future.